app/src/app/pages/shop/shared/header/header.component.html

```diff
@@ -0,0 +1,17 @@
+<header class="header">
+  <div class="container">
+    <nav class="header_navigation">
+      <ul class="header_list">
+        <li [routerLink]="['/']" class="header_list_item header_logo">
+          <a   class="header_list_link logo"></a>
+        </li>
+        <li class="header_list_item">
```

Comment:
you can create a mock for links as an array and after that use the ngFor directive.